lib/ddtrace.rb

```diff
@@ -14,6 +16,10 @@ module Datadog
   def self.tracer
     @tracer
   end
+
+  def self.tracer=(_tracer)
+    @tracer = tracer
```

Comment:
this action if exposed to users may be dangerous. Rails by default uses the global tracer (so ``Datadog.tracer``) and if the tracer is changed after Rails initialization, we may have two different tracers in our code. Just to point that out.